Samples/PushNotifications/cpp-console-unpackaged/cpp-console-unpackaged.cpp

```diff
@@ -0,0 +1,206 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include <windows.h>
+#include <iostream>
+
+#include <appmodel.h>
+#include <wil/result.h>
+#include <wil/cppwinrt.h>
+
+#include <winrt/Windows.ApplicationModel.Activation.h>
+#include <winrt/Windows.ApplicationModel.Background.h>
+#include <winrt/Windows.Foundation.h>
+#include <winrt/Microsoft.Windows.AppLifecycle.h>
+#include <winrt/Microsoft.Windows.PushNotifications.h>
+#include <winrt/Windows.Globalization.DateTimeFormatting.h>
+
+#include <MddBootstrap.h>
+
+using namespace winrt::Microsoft::Windows::AppLifecycle;
+using namespace winrt::Microsoft::Windows::PushNotifications;
+using namespace winrt::Windows::ApplicationModel::Background; // BackgroundTask APIs
+using namespace winrt::Windows::Foundation;
+using namespace winrt::Windows::Globalization::DateTimeFormatting;
+
+// To obtain an AAD RemoteIdentifier for your app,
+// follow the instructions on https://docs.microsoft.com/azure/active-directory/develop/quickstart-register-app
```

Comment:
Update this point to the current https://docs.microsoft.com/en-us/windows/apps/windows-app-sdk/notifications/push/push-quickstart#configure-your-apps-identity-in-azure-active-directory